Tests/PersistenceHandler/RabbitMqPersistenceHandlerTest.php

```diff
@@ -290,10 +314,19 @@ public function testPersistRetrieveValidDataFromPdfFile()
 
         $persistenceHandler = new RabbitMqPersistenceHandler($queue, $pdfParser);
 
-        $this->assertNull($persistenceHandler->persist($resource));
+        $crawlJob = $this
+            ->getMockBuilder('Simgroep\ConcurrentSpiderBundle\CrawlJob')
+            ->setMethods(null)
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        $this->assertNull($persistenceHandler->persist($resource, $crawlJob));
     }
 
-    public function testPersistRetrieveValidDataFromWebPageWithAllDefaultValuesInFields()
+    /**
+     * @Test
```

Comment:
change name of function to begin with test in name so we have everything a consistency